core/typeguards.ts

```diff
@@ -0,0 +1,60 @@
+import is from '@sindresorhus/is';
+import { validate as uuidValidate } from 'uuid';
+import { version as uuidVersion } from 'uuid';
```

Comment:
Codacy has a fix for the issue: [Replace `'uuid'` with `"uuid"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7711626)
```suggestion
import { version as uuidVersion } from "uuid";
```